azimuth/modules/word_analysis/top_words.py

```diff
@@ -19,21 +19,25 @@
     TopWordsResult,
 )
 from azimuth.utils.dataset_operations import get_predictions_from_ds
-from azimuth.utils.ml.third_parties.stop_words import STOP_WORDS
 from azimuth.utils.project import saliency_available
+from azimuth.utils.utterance import clean_utterance
 
 MIN_SALIENCY = 0.01
 
 
-class TopWordsModule(FilterableModule[ModelContractConfig]):
-    """Returns the most important words in terms of their saliency value or frequency."""
+class TopWordsModule(FilterableModule[AzimuthConfig]):
+    """Returns the most important words in terms of their saliency value or frequency.
+
+    Note: The config scope is AzimuthConfig because the module relies on both the pipeline or the
+    syntax config, depending if saliency is available. AzimuthConfig is a bit too broad, but it
+    should not be a problem since this module computes fast.
```

Comment:
So minor and picky, feel free to take or leave. Super helpful docstring note, though.
```suggestion
    Note: The config scope is AzimuthConfig because the module relies on both the pipeline or the
    syntax config, depending on whether saliency is available. AzimuthConfig is a bit too broad,
    but it should not be a problem since this module computes quickly.
```